app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
+    title: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 300,
```

Comment:
No specific reason -- I thought I had seen 300 characters for maximum length for a title in other websites. However I can change this to 200 if it matches with the OSF's style.